seed.js

```diff
@@ -30,22 +30,22 @@ const users = [
   },
 ];
 
-// const sessions = [
-//   { sessionId: `SID=${Date.now()}`},
-//   { sessionId: `SID=${Date.now() * 1500}`},
-// ];
+const sessions = [
+  { sessionId: `SID=${Date.now()}`},
+  { sessionId: `SID=${Date.now() * 1500}`},
+];
 
-// const orderProducts = [
-//   { productQuantity: 2 },
-//   { productQuantity: 3 },
-//   { productQuantity: 1 },
-//   { productQuantity: 2 },
-// ]
+const orderProducts = [
```

Comment:
This is still a bit weird...

Look here: https://sequelize.org/master/class/lib/associations/belongs-to-many.js~BelongsToMany.html

Id imagine what you want to do is:

```javascript
const order = await Order.create({
  orderTotal: 30,
  status: 'COMPLETE',
});
const product = await Product.create({
    name: 'Edward',
    description: `Edward (1917-2002) is our grandfather, Hill Farmstead Brewery rests upon the land that was once home to him and his five children. In his honor, this American Pale Ale is dutifully crafted from American malted barley, a plethora of American hops, our ale yeast and water from Edward’s well. It is unfiltered and dry hopped. Aromatic and flowery, with impressions of citrus and pine, this is the ale that I dream to have shared with Edward. Pale and Caramel malt, Centennial, Chinook, Columbus, Simcoe, and Warrior hops, House Ale Yeast, and our Well Water.`,
    imageUrl: 'https://i.imgur.com/oayGuSE.png',
    price: 8.0,
});

await order.addProduct(product, { through: { productQuantity: 1 } });
```

Which looks good to me. Also, if you were unaware, you can also **eager create**.

```javascript
const fullOrder = await Order.create({
  orderTotal: 30,
  status: 'COMPLETE',
  products: [
    {
      name: 'Edward',
      description: `Edward (1917-2002) is our grandfather, Hill Farmstead Brewery rests upon the land that was once home to him and his five children. In his honor, this American Pale Ale is dutifully crafted from American malted barley, a plethora of American hops, our ale yeast and water from Edward’s well. It is unfiltered and dry hopped. Aromatic and flowery, with impressions of citrus and pine, this is the ale that I dream to have shared with Edward. Pale and Caramel malt, Centennial, Chinook, Columbus, Simcoe, and Warrior hops, House Ale Yeast, and our Well Water.`,
      imageUrl: 'https://i.imgur.com/oayGuSE.png',
      price: 8.0,
      orderProduct: {
        productQuantity: 1,
      },
    }
  ],
});
```

But I think arguably that may look worse. My end point here is that **we shouldn't be manually creating rows in a join table.**